nyan/main.py

```diff
@@ -0,0 +1,45 @@
+"""Nyan Cat Animation! Rotate the screen with 'A'."""
+
+___name___         = "nyan"
+___license___      = "MIT"
+___dependencies___ = ["sleep", "app", "ugfx_helper", 
+                      "shared/nyan/0.png",
+                      "shared/nyan/1.png",
+                      "shared/nyan/2.png",
+                      "shared/nyan/3.png",
+                      "shared/nyan/4.png",
+                      "shared/nyan/5.png",
+                      "shared/nyan/6.png",
+                      "shared/nyan/7.png",
+                      "shared/nyan/8.png",
+                      "shared/nyan/9.png",
+                      "shared/nyan/10.png",
+                      "shared/nyan/11.png"]
+                      
+___categories___   = ["Homescreens", "Other"]
+
+import ugfx_helper, os, wifi, ugfx, http, time, sleep, app
+from tilda import Buttons
+
+# initialize screen
+ugfx_helper.init()
+ugfx.clear(ugfx.BLACK)
+
+ugfx.backlight(100)
+
+n = 0
+r = 270
+while True:
+    ugfx.display_image( 0, 90, "shared/nyan/{}.png".format(n) )
+    n = (n+2) % 12
```

Comment:
This skips every other frame, did you mean `n+1`?

Reply:
Well, showin the images is a bit slow, so playing every second frame was a kind of a hack for the last evening in the camp to make the animation a bit faster :)
I'm exploring the possibilities of optimizing the animation to make it run with the intended speed in the future.

Reply:
"Fun :) Looks like each png file is duplicated - 0 & 6 are the same, 1 & 7, etc." - yes, thank you for pointing out! They were the same - the gif export app lied to me!
(removed)